Translations update from Weblate

ubuntukylin-weblate opened the pull request:
Translations update from [Weblate](https://weblate.ubuntukylin.com/projects/ukui-power-manager/ukui-power-manager/) for ukui-power-manager/ukui-power-manager.



Current translation status:

![Weblate translation status](https://weblate.ubuntukylin.com/widgets/ukui-power-manager/-/ukui-power-manager/horizontal-auto.svg)

weblate committed:
Update translation files

Updated by "Update LINGUAS file" hook in Weblate.

Translation: ukui-power-manager/ukui-power-manager
Translate-URL: https://weblate.ubuntukylin.com/projects/ukui-power-manager/ukui-power-manager/

brsvh commented:
Just close this pr with not merge.

Weblate auto generate LINGUAS file and create pull request when import this repositroy. Maybe this project have some duplicated gettext file cause mistake when generate .

lxwndr committed:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (449 of 449 strings)

Translation: ukui-power-manager/ukui-power-manager
Translate-URL: https://weblate.ubuntukylin.com/projects/ukui-power-manager/ukui-power-manager/pt_BR/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (449 of 449 strings)

Translation: ukui-power-manager/ukui-power-manager
Translate-URL: https://weblate.ubuntukylin.com/projects/ukui-power-manager/ukui-power-manager/fr/

Commit:
Translated using Weblate (French)

Currently translated at 100.0% (449 of 449 strings)

Translation: ukui-power-manager/ukui-power-manager
Translate-URL: https://weblate.ubuntukylin.com/projects/ukui-power-manager/ukui-power-manager/fr/

hugok79 committed:
Translated using Weblate (Portuguese)

Currently translated at 100.0% (449 of 449 strings)

Translation: ukui-power-manager/ukui-power-manager
Translate-URL: https://weblate.ubuntukylin.com/projects/ukui-power-manager/ukui-power-manager/pt/

Commit:
Translated using Weblate (Sinhala)

Currently translated at 11.5% (52 of 449 strings)

Translation: ukui-power-manager/ukui-power-manager
Translate-URL: https://weblate.ubuntukylin.com/projects/ukui-power-manager/ukui-power-manager/si/
